qa-tests-backend/src/test/groovy/ImageSignatureVerificationTest.groovy

```diff
@@ -226,6 +237,12 @@ QC+pUMTUP/ZmrvmKaA+pi55F+w3LqVJ17zwXKjaOEiEpn/+lntl/ieweeQ==
             assert policyID
             CREATED_POLICY_IDS.add(policyID)
         }
+
+        // Reassessing policies will trigger a re-enrichment of images.
+        PolicyService.reassessPolicies()
+
+        // Allow reassess policies to finish.
+        sleep 5000
```

Comment:
:( 
Is there any way to check if it's already done. Why 5s?

Reply:
Sadly not really, but I've done the following now: instead of checking whether the image metadata has been fetched in the `setup` I moved it after the `setupSpec` with a polling. This will get rid of the sleep and wait for reassess to finish properly.